src/main/java/core/basesyntax/AreaCalculationable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculationable {

```

Comment:
```suggestion
public interface AreaCalculator {
```
```suggestion
public interface AreaCalculable {
```
nouns for interfaces also work